org.eclipse.mylyn.releng/oomph/Mylyn.setup

```diff
@@ -92,10 +94,60 @@
     <setupTask
         xsi:type="pde:APIBaselineTask"
         id="mylyn.baseline"
+        disabled="true"
         name="Mylyn"
         version="3.25"
         location="${workspace.location/../api.baseline}"
         remoteURI="https://download.eclipse.org/mylyn/drops/3.25.0/mylyn-3.25.0-api.zip"/>
+    <setupTask
+        xsi:type="setup.targlets:TargletTask"
+        id="mylyn.target.api"
+        targetName="Mylyn Modular Target for API"
+        activateTarget="false">
+      <targlet
+          name="Mylyn"
+          includeSources="false">
+        <requirement
+            name="org.eclipse.mylyn_feature.feature.group"/>
+        <requirement
+            name="org.eclipse.mylyn.context_feature.feature.group"/>
+        <requirement
+            name="org.eclipse.mylyn.trac_feature.feature.group"/>
+        <requirement
+            name="org.eclipse.mylyn.discovery.feature.group"/>
+        <requirement
+            name="org.eclipse.mylyn.ide_feature.feature.group"/>
+        <requirement
+            name="org.eclipse.mylyn.java_feature.feature.group"/>
+        <requirement
+            name="org.eclipse.mylyn.pde_feature.feature.group"/>
+        <requirement
+            name="org.eclipse.mylyn.team_feature.feature.group"/>
+        <requirement
+            name="org.eclipse.mylyn.bugzilla_feature.feature.group"/>
+        <requirement
+            name="org.eclipse.mylyn.git.feature.group"/>
+        <requirement
+            name="org.eclipse.cdt.mylyn.feature.group"/>
+        <requirement
+            name="org.eclipse.mylyn.github.feature.feature.group"/>
+        <repositoryList
+            name="latest">
+          <repository
+              url="https://download.eclipse.org/mylyn/releases/3.26"/>
+          <repository
+              url="https://download.eclipse.org/egit/updates-6.2"/>
+          <repository
+              url="https://download.eclipse.org/tools/cdt/releases/11.0"/>
+          <repository
+              url="https://download.eclipse.org/eclipse/updates/4.27/R-4.27-202303020300"/>
```

Comment:
actually we had Eclipse Platform 4.26 in our target, it is better to use the target from the latest release to define the baseline for upcoming one